web/app/view/edit/ComputedAttributes.js

```diff
@@ -46,13 +46,29 @@ Ext.define('Traccar.view.edit.ComputedAttributes', {
             dataIndex: 'description'
         }, {
             text: Strings.sharedAttribute,
-            dataIndex: 'attribute'
+            dataIndex: 'attribute',
+            renderer: function (value) {
+                var attribute = Ext.getStore('PositionAttributes').getById(value);
+                if (attribute) {
+                    return attribute.get('name');
+                } else {
+                    return value;
+                }
+            }
         }, {
             text: Strings.sharedExpression,
             dataIndex: 'expression'
         }, {
             text: Strings.sharedType,
-            dataIndex: 'type'
+            dataIndex: 'type',
+            renderer: function (value) {
```

Comment:
Similar code seems to be duplicated in many places. I suggest to create a method for it somewhere to avoid duplication.